src/vibrator-sysfs.cpp

```diff
@@ -18,27 +18,65 @@
 
 #include "vibrator-sysfs.h"
 
+#include <fstream>
+#include <string>
 #include <iostream>
 #include <unistd.h>
 
 // if the kernel support led triggers
 // we set 
 namespace hfd {
 
+// True iff the standard transient trigger is supported.
```

Comment:
A typo

Reply:
'iff' is short for 'if and only if', I'll rewrite it to be less confusing :sweat_smile: